app/scripts/modules/google/src/loadBalancer/configure/http/healthCheck/healthChecks.html

```diff
@@ -3,10 +3,10 @@
     <div class="alert alert-warning">
       <p><i class="fa fa-exclamation-triangle"></i>
         The following health checks have not been associated with a backend service:
-        <ul>
-          <li ng-repeat="service in ctrl.command.getUnusedHealthChecks(ctrl.command)" ng-bind-html="service"></li>
-        </ul>
       </p>
+      <ul>
+        <li ng-repeat="service in ctrl.command.getUnusedHealthChecks(ctrl.command)" ng-bind-html="service"></li>
+      </ul>
```

Comment:
This p tag might be applying styles to the ul that we're losing here, worth a look

Reply:
I'm not worried about it.  `<ul>` is not legal inside a `<p>` tag (only inline content is legal).  

When the browser parses this, it puts the `<ul>` as a peer to (not inside of) the  `<p>` tag.

<img width="632" alt="screen shot 2019-02-21 at 4 50 41 pm" src="https://user-images.githubusercontent.com/2053478/53212289-50309200-35f9-11e9-923d-4dfd8aeecc7a.png">

Reply:
Look at that! Thanks for the tip.